bindings/twilio/sendgrid/sendgrid.go

```diff
@@ -102,25 +106,43 @@ func (sg *SendGrid) Invoke(ctx context.Context, req *bindings.InvokeRequest) (*b
 	// We allow two possible sources of the properties we need,
 	// the component metadata or request metadata, request takes priority if present
 
+	// Set the email from name, this is optional
+	fromName := ""
+	if sg.metadata.EmailFromName != "" {
+		fromName = sg.metadata.EmailFromName
+	}
+	if req.Metadata["emailFromName"] != "" {
+		fromName = req.Metadata["emailFromName"]
+	}
+
 	// Build email from address, this is required
 	var fromAddress *mail.Email
 	if sg.metadata.EmailFrom != "" {
-		fromAddress = mail.NewEmail("", sg.metadata.EmailFrom)
+		fromAddress = mail.NewEmail(fromName, sg.metadata.EmailFrom)
 	}
 	if req.Metadata["emailFrom"] != "" {
-		fromAddress = mail.NewEmail("", req.Metadata["emailFrom"])
+		fromAddress = mail.NewEmail(fromName, req.Metadata["emailFrom"])
 	}
 	if fromAddress == nil {
 		return nil, fmt.Errorf("error SendGrid from email not supplied")
 	}
 
+	// Set the email to name, this is optional
+	toName := ""
```

Comment:
Same as per my other comment

Reply:
I will make that change

Reply:
@ItalyPaleAle I made the suggested change.